t/file.t

```diff
@@ -23,20 +23,21 @@ if ( $^O =~ m/^MSWin/ ) {
   $tmp_dir = $ENV{TMP};
 }
 
-my $filename = "$cwd/test-$$.txt";
+my $filename = "$cwd/test-{a,b}-$$.txt";
```

Comment:
Kudos for the testing! :+1: Would it be enough to test only the newly added "glob" case?